src/Sulu/Component/Rest/ListBuilder/Expression/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Expression\Exception;
+
+/**
+ * Exceptions is throw when an invalid parameter is passed to an expression.
+ */
+class InvalidArgumentException extends ExpressionException
```

Comment:
Would rename to  `InvalidExpressionArgumentException`, could be confused with the standard exception too easily otherwise.